packages/azure-themes/src/azure/AzureThemeHighContrastLight.ts

```diff
@@ -0,0 +1,138 @@
+  rowBorder: HighContrastLightSemanticColors.radioButton.circle.checkedDisabled,
+  toggleDisabledBackground: HighContrastLightSemanticColors.toggle.disabled.backrgound,
+  variantBorder: HighContrastLightSemanticColors.controlOutlines.rest,
+  // extended
+  controlAccent: HighContrastLightSemanticColors.controlOutlines.accent,
+  controlOutline: HighContrastLightSemanticColors.controlOutlines.rest,
+  controlOutlineDisabled: HighContrastLightSemanticColors.controlOutlines.disabled,
+  controlOutlineHovered: HighContrastLightSemanticColors.controlOutlines.hover,
+  iconButtonBackground: HighContrastLightSemanticColors.primaryButton.hover.background,
+  iconButtonFill: HighContrastLightSemanticColors.text.icon,
+  iconButtonFillHovered: HighContrastLightSemanticColors.primaryButton.hover.text,
+  labelText: HighContrastLightSemanticColors.text.label,
+  statusErrorBackground: HighContrastLightSemanticColors.statusBar.error,
+  statusErrorText: HighContrastLightSemanticColors.text.body,
+  statusErrorIcon: CommonSemanticColors.icons.error,
+  statusInformationBackground: HighContrastLightSemanticColors.statusBar.information,
+  statusInformationText: HighContrastLightSemanticColors.text.body,
+  statusInformationIcon: CommonSemanticColors.icons.information,
+  statusSuccessBackground: HighContrastLightSemanticColors.statusBar.okay,
+  statusSuccessText: HighContrastLightSemanticColors.text.body,
+  statusSuccessIcon: CommonSemanticColors.icons.okay,
+  statusWarningBackground: HighContrastLightSemanticColors.statusBar.warning,
+  statusWarningText: HighContrastLightSemanticColors.text.body,
+  statusWarningIcon: CommonSemanticColors.icons.warning,
+  textFieldBorderDisabled: HighContrastLightSemanticColors.text.disabled,
+  textFieldBackgroundDisabled: HighContrastLightSemanticColors.disabledButton.background,
+
+  // temporary work around for high contrast themes
+  choiceGroupContainerBorder: '1px',
+  choiceGroupContainerBorderStyle: 'dashed',
+  listUnderline: 'underline',
+  linkBorderStyle: 'dashed',
+};
+
+export const AzureThemeHighContrastLight: ITheme = createTheme({
+  fonts: {
+    medium: {
+      fontFamily: StyleConstants.fontFamily,
+      fontSize: FontSizes.size13,
+    },
+  },
+  palette: {
+    themePrimary: HighContrastLightSemanticColors.controlOutlines.accent,
+    neutralPrimary: HighContrastLightSemanticColors.text.body,
+    neutralDark: HighContrastLightSemanticColors.text.body,
+    neutralLight: HighContrastLightSemanticColors.shimmer.secondary, // shimmer elements
+    neutralLighter: HighContrastLightSemanticColors.shimmer.primary, // shimmer elements
+    neutralLighterAlt: HighContrastLightSemanticColors.item.hover, // nav highlight
+    neutralQuaternaryAlt: HighContrastLightSemanticColors.item.select, // expand button on list controls
+    neutralSecondary: HighContrastLightSemanticColors.text.label, // persona
```

Comment:
what do you mean by 'persona' here? elaborate more in the comment so others can understand better?